File: Day 2/9.cpp
//SELF KIN
bool isPalindrome(int x){
    if(x<0||(x%10)==0&&x!=0) return 0;
    unsigned int new = 0;
    while(x>new)
    {
        new = (new*10) + (x%10);
        x/=10;
    }
    if(new==x||new/10==x) return 1;
    return 0;
}

File: Day 2/9.java
//SELF PALAK

class Solution {
    public boolean isPalindrome(int x) {
        if(x<0)
            return false;
        else{
        int i;
        int r=0,d=0;
        for(i=x;i!=0;i=i/10)
        {
            d=i%10;
            r=r*10+d;
        }
        System.out.println(r);
        if(r==x)
            return true;
        else return false;
    }
    }
}
